app/src/main/java/com/losextraditables/bu/instagrammers/view/holder/UserViewHolder.java

```diff
@@ -2,37 +2,60 @@
 
 
 import android.content.Context;
+import android.graphics.Color;
+import android.graphics.PorterDuff;
 import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
 import android.view.View;
 import android.widget.ImageView;
 import android.widget.TextView;
 
+import com.daimajia.androidanimations.library.Techniques;
+import com.daimajia.androidviewhover.BlurLayout;
 import com.losextraditables.bu.R;
 import com.losextraditables.bu.instagrammers.view.model.InstagrammerModel;
 import com.squareup.picasso.Picasso;
 
 import butterknife.Bind;
 import butterknife.ButterKnife;
+import de.hdodenhof.circleimageview.CircleImageView;
 
 public class UserViewHolder extends RecyclerView.ViewHolder {
 
     private final Context context;
-    @Bind(R.id.instagrammer_picture)
+    @Bind(R.id.user_blur_image)
```

Comment:
would be possible for you to maintain the naming convention? I mean, call the resource something like instagrammer_blur_image. They're not users, but instagrammers in our conceptual model you know.